dev/tasks/python-wheels/github.linux.yml

```
@@ -60,15 +63,18 @@ jobs:
         env:
           GITHUB_TOKEN: {{ '${{ secrets.GITHUB_TOKEN }}' }}
         run: |
-          if [ "{{ manylinux_version }}" = "2014" ] && [ "{{ arch }}" = "arm64" ]; then
+          if [ "{{ linux_wheel_version }}" = "2014" ] && [ "{{ arch }}" = "arm64" ]; then
             # We can't use NuGet on manylinux2014_aarch64 because Mono is old.
             :
+          elif [ "{{ linux_wheel_kind }}" = "musllinux" ] && [ "{{ arch }}" = "arm64" ]; then
+            # We can't use NuGet on musl arm systems because it fails to build abseil:x64-linux
```

Is this comment correct?
It's strange that `:x64-linux` is built with `musllinux` + `arm64`.

---

Yup, when I tried to use NuGet it strangely fails to build a x64 binary. I don't know why it tries that instead of a arm64 version. Please check my previous comment [here](https://github.com/apache/arrow/pull/45470#issuecomment-2679968849).

---

Hmm. I don't remember whether we have correct configuration when the comment is written...

How about the following?

```suggestion
            # We can't use NuGet on musl arm systems because the official NuGet binary for arm doesn't exist.
            # See also: https://www.nuget.org/downloads
```